src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLoggedOutUser.java

```diff
@@ -39,6 +39,9 @@ public void loggedOutUserCanCreateNewWiki() {
     new WikiFactory().setIsTestWiki(article.getWikiID(), true);
 
     article.verifyWikiTitleHeader(wikiName);
-    article.verifyUserLoggedIn(credentials.userName10);
+    // Caused by: https://wikia-inc.atlassian.net/browse/PLATFORM-3829
+    // User wont be logged in after the migration is finished, since we are not going to sync access token over in CNW
+    // TODO: Enable once wikia.com -> fandom.com migration is complete
+    // article.verifyUserLoggedIn(credentials.userName10);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)